pkg/cnb/project_descriptor.go

```diff
@@ -23,6 +24,13 @@ func ProcessProjectDescriptor(appDir, platformDir string) error {
 	if err != nil {
 		return err
 	}
+	if d.Build.Buildpacks != nil {
+		logger.Println("warning: buildpacks provided in project descriptor file will be ignored")
```

Comment:
I think @samj1912 was suggesting `info` as a message because it is may not be a `warning` situation?

Reply:
I guess my only concern was elevating the message to the user because this may be unexpected behavior, i.e. if they explicitly provided builder/buildpacks in the descriptor expecting them to be consumed I wanted to call it out a little more aggressively.

But you're right it would still probably be an INFO level message not a WARN.

Reply:
We just had a discussion around project descriptor portability across different platforms in the CNB WG meeting today. I think we (CNB) want to define some sort of guidance for what platforms are expected to do when they find project descriptor or features from it they can't/don't want to support.

Suggestions in the CNB WG meeting ranged from straight up build errors for unsupported features or warnings. 
---

The reason I had suggested info as the log level instead of warning is because in the pack world, flags that are explicitly passed in through the CLI override the project descriptor and I believe that the resource definition is the closest equivalent to explicit flags here in the kpack world. If the user explicitly passed in a builder, that should override the builder in the project toml.

Looking back at it, I think we may want to warn for the buildpack key but keep it to an info level for the builder key?

Thoughts?

Reply:
The resource definition explicit flag equivalence makes sense to me. I personally don't love differing the levels of the two log messages though. Would rather keep them in sync because it could give users the wrong impression of severity, like buildpacks being ignored is somehow more severe than builders being ignored and neither are *actually* problems technically.

That being said would would be happy to drop both to `info` level if that is good with you!